src/DynamoPublish/DynamoPublishExtension.cs

```diff
@@ -111,29 +118,112 @@ private MenuItem GenerateMenuItem()
             var isEnabled = loadedParams.CurrentWorkspaceModel is HomeWorkspaceModel && startupParams.AuthProvider != null;
             item.IsEnabled = isEnabled;
 
-            item.Click += (sender, args) =>
+            // Open the configuration file using the dll location.
+            var config = ConfigurationManager.OpenExeConfiguration(typeof(PublishModel).Assembly.Location);
+            // Get the appSettings section.
+            var appSettings = (AppSettingsSection)config.GetSection("appSettings");
```

Comment:
We should check if this is present and crash with a more descriptive error if this is not present.